android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -106,6 +106,9 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
           !it.value.isNullOrEmpty() &&
           !it.value.contains(STRING_INTERPOLATION_PREFIX)
       }
+    questionnaireViewModel.setFamilyLogicalIdActionParam(
+      actionParams.firstOrNull { it.paramType == ActionParameterType.UPDATEABLEIDS }
```

Comment:
I do not think we need this implementation. Update the filter logic for action Parameters above to include `ActionParameterType.UPDATEABLEIDS` instead.